src/llvm/ext/llvm_ext.cc

```diff
@@ -6,6 +6,10 @@
 #include <llvm/IR/DebugLoc.h>
 #include <llvm/IR/LLVMContext.h>
 #include <llvm/IR/Metadata.h>
+#include "llvm/Analysis/ModuleSummaryAnalysis.h"
```

Comment:
It's the same (in this context). I need to add some ifdef too for older llvm, I will change it then.